frontend/packages/kubevirt-plugin/integration-tests-cypress/view/vm.ts

```diff
@@ -39,11 +39,14 @@ export const vm = {
       sshEnable,
       startOnCreation,
       template,
+      sourceAvailable,
     } = vmData;
     virtualization.vms.visit();
     wizard.vm.open();
     wizard.vm.processSelectTemplate(template);
-    wizard.vm.processBootSource(provisionSource, cdrom, pvcSize, pvcName, pvcNS);
+    if (!sourceAvailable || sourceAvailable === undefined) {
```

Comment:
`undefined` should evaluate to false, so I think you can remove the `|| sourceAvailable === undefined`.

Reply:
changed.